GitUI/Avatars/AvatarMemoryCache.cs

```diff
@@ -40,12 +39,15 @@ public async Task<Image> GetAvatarAsync(string email, string name, int imageSize
 
             var image = await _inner.GetAvatarAsync(email, name, imageSize);
 
-            lock (_cache)
+            if (image != null)
             {
-                _cache.Add((email, imageSize), image);
-
-                return image;
+                lock (_cache)
+                {
+                    _cache.Add((email, imageSize), image);
+                }
             }
+
+            return image;
```

Comment:
This fixes a bug where we could attempt to add a null image into the cache, which results in an exception.

Reply:
`if (image is not null)`

Reply:
@gerhardol updated.